src/PlaywrightSharp.Tests/ElementHandleClickTests.cs

```diff
@@ -112,7 +112,7 @@ public async Task ShouldDoubleClickTheButton()
             }");
 
             var button = await Page.QuerySelectorAsync("button");
-            await button.DblClickAsync();
+            await button.DblclickAsync();

```

Comment:
This needs to be fixed in upstream.

Reply:
https://github.com/microsoft/playwright/pull/5899